Makefile

```diff
@@ -169,12 +172,12 @@ bootstrap-destroy: ## Destroy bootstrap
 bosh-cli: ## Create interactive connnection to BOSH container
 	concourse/scripts/bosh-cli.sh
 
-.PHONY: pipelines
+.PHONY: pipelines check-aws-credentials
 pipelines: ## Upload pipelines to Concourse
```

Comment:
I guess you meant to add `check-aws-credentials` as a dependency of this task, in which case it needs to be on the line below.